src/main/interface/settings.c

```diff
@@ -618,7 +618,7 @@ const clivalue_t valueTable[] = {
 // PG_CURRENT_SENSOR_ADC_CONFIG
 #ifdef USE_VIRTUAL_CURRENT_METER
     { "ibatv_scale",                VAR_INT16  | MASTER_VALUE, .config.minmax = { -16000, 16000 }, PG_CURRENT_SENSOR_VIRTUAL_CONFIG, offsetof(currentSensorVirtualConfig_t, scale) },
-    { "ibatv_offset",               VAR_INT16  | MASTER_VALUE, .config.minmax = { -16000, 16000 }, PG_CURRENT_SENSOR_VIRTUAL_CONFIG, offsetof(currentSensorVirtualConfig_t, offset) },
+    { "ibatv_offset",               VAR_UINT16 | MASTER_VALUE, .config.minmax = { 0, 16000 }, PG_CURRENT_SENSOR_VIRTUAL_CONFIG, offsetof(currentSensorVirtualConfig_t, offset) },
```

Comment:
Was there a problem with allowing negative offsets? Seems like it might be useful to the user if the virtual current meter was over calculating.

Reply:
For the virtual current meter, the 'offset' is really the constant consumption of the craft when disarmed (in mA):

https://github.com/betaflight/betaflight/blob/eecb59db4542cbf577bedf853d8b314422825dae/src/main/sensors/current.c#L187-L196

Having a negative offset here does not make much sense, as this will mean that the current becomes negative, and the consumption decreases when the craft is disarmed.